build.sbt

```diff
@@ -16,99 +16,66 @@ lazy val root = project
   .in(file("."))
   .settings(name := "bigquery-loader")
   .aggregate(common, loader, streamloader, mutator, repeater)
+  .aggregate(loaderDistroless, streamloaderDistroless, mutatorDistroless, repeaterDistroless)
   .settings(assembly / aggregate := false)
 // format: on
 
 lazy val common = project
   .in(file("modules/common"))
   .enablePlugins(BuildInfoPlugin)
   .settings(BuildSettings.commonBuildSettings)
-  .settings(
-    libraryDependencies ++= Seq(
-      Dependencies.Beam.bigQuery,
-      Dependencies.cats,
-      Dependencies.catsEffect,
-      Dependencies.circe,
-      Dependencies.circeConfig,
-      Dependencies.circeJawn,
-      Dependencies.circeLiteral,
-      Dependencies.circeParser,
-      Dependencies.decline,
-      Dependencies.fs2,
-      Dependencies.logging,
-      Dependencies.slf4j,
-      Dependencies.analyticsSdk,
-      Dependencies.badrows,
-      Dependencies.igluClient,
-      Dependencies.jacksonDatabind,
-      Dependencies.igluCoreCirce,
-      Dependencies.schemaDdl,
-      Dependencies.specs2,
-      Dependencies.nettyCodec,
-      Dependencies.nettyCodecHttp,
-      Dependencies.nettyCodecHttp2,
-      Dependencies.sentry
-    )
-  )
 
 lazy val loader = project
   .in(file("modules/loader"))
-  .enablePlugins(BuildInfoPlugin)
-  .enablePlugins(JavaAppPackaging)
+  .enablePlugins(BuildInfoPlugin, JavaAppPackaging)
   .settings(BuildSettings.loaderBuildSettings)
```

Comment:
Maybe `import BuildSettings._` can make it even more concise?

Reply:
I generally dislike importing `Something._` because I find it adds too much mystery about where names are coming from.  I value clarity over conciseness when I'm reading a file.

My exception is when importing implicits, where I find `import something.implicits._` convenient and clear enough.